test/unit/gateways/stripe_payment_intents_test.rb

```diff
@@ -55,6 +55,16 @@ def test_successful_create_and_update_intent
     assert_equal 'requires_confirmation', update.params['status']
   end
 
+  def test_contains_statement_descriptor_suffix
+    options = @options.merge(capture_method: 'manual', statement_descriptor_suffix: 'suffix')
+
+    stub_comms(@gateway, :ssl_request) do
+      @gateway.create_intent(@amount, @visa_token, options)
+    end.check_request do |method, endpoint, data, headers|
+      assert_match(/statement_descriptor_suffix=suffix/, data)
```

Comment:
Ah, _this_ must be the pattern that [this code](https://github.com/activemerchant/active_merchant/pull/3527/files#diff-89a3b27198f4d2243c901979d241a75cL44-L47) was trying to follow. Good to know!